packages/api-server/src/methods/modules/eth.ts

```diff
@@ -1128,25 +1163,38 @@ export class Eth {
   async sendRawTransaction(args: [string]): Promise<Hash> {
     try {
       const data = args[0];
-      const rawTx = await generateRawTransaction(data, this.rpc);
+      const [rawTx, autoCreateCacheKeyAndValue] = await generateRawTransaction(
+        data,
+        this.rpc
+      );
```

Comment:
The `autoCreateCacheKeyAndValue` is passed along: `sendRawTransaction <= generateRawTransaction <= parseRawTransactionData`.

I personally think the code is  too coupling.

Reply:
Need to set `from_id = 0` instead of throw error here.